internal/report/report.go

```diff
@@ -432,6 +432,10 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 		}
 	}
 
+	if len(syms) == 0 {
+		return fmt.Errorf("no matches found for regexp: %v", o.Symbol)
```

Comment:
Here and in other error messages: %v -> %s? Using narrower format specifier might help catch mistakes (such as passing a wrong variable).